docs/rfcs/0001-high-level-architecture.md

```diff
@@ -112,64 +112,64 @@ The following values apply to the whole system, and are sorted roughly in priori
 
 > **Disclaimer:** All of these values are aspirational, they are not literal guarantees that can be used for liability claims. We expect to iteratively improve towards and get closer to these goals as the project matures and new versions are released.
 
-1. **Security**
+1. <span id="value-security"></span> **Security**
    1. **Security by design**: Security should be at the top of our minds at every decision we make. All design proposals must consider how the proposed change affects security concerns.
```

Comment:
I would also add spans to all of these; IMO they are important to reference separately as well

Reply:
Sure, it's going add a bit of clutter but I can do that.

Reply:
Done